src/SFA.DAS.LevyTransferMatching/InnerApi/LevyTransferMatching/Requests/CreateApplicationRequest.cs

```diff
@@ -42,5 +42,10 @@ public class CreateApplicationRequestData
         public string BusinessWebsite { get; set; }
         public string UserId { get; set; }
         public string UserDisplayName { get; set; }
+        public bool MatchSector { get; set; }
```

Comment:
See other comments about this being an unexpected change.

Reply:
changes made as requested